docs/tutorial/part06.rst

```diff
@@ -2,5 +2,440 @@
+                'sourceID': self.sourceID,
+                'address': self.address,
+                'jurisdiction': self.jurisdiction,
+                'service_provider': self.service_provider,
+                'countries': self.countries,
+                'jurisdiction_description': self.jurisdiction_description,
+                'valid_until': self.valid_until,
+                'ibcRUC': self.ibcRUC,
+                'name': self.name,
+                'country_codes': self.country_codes,
+                'incorporation_date': self.incorporation_date,
+                'node_id': self.node_id,
+                'status': self.status,
+            },
+        }
+
+If you look at this closely you'll notice this is just all the properties on the ``Entity`` class mapped to a dictionary but the relationships, we'll explain later why we don't serialize the relationships here and how to do it, you will need to do the same for the other model classes:
+
+Address::
+
+    @property
+    def serialize(self):
+        return {
+            'node_properties': {
+                'sourceID': self.sourceID,
+                'country_codes': self.country_codes,
+                'valid_until': self.valid_until,
+                'address': self.address,
+                'countries': self.countries,
+                'node_id': self.node_id,
+            },
+        }
+
+Intermediary::
+
+    @property
+    def serialize(self):
+        return {
+            'node_properties': {
+                'sourceID': self.sourceID,
+                'valid_until': self.valid_until,
+                'name': self.name,
+                'country_codes': self.country_codes,
+                'countries': self.countries,
+                'node_id': self.node_id,
+                'status': self.status,
+            },
+        }
+
+Officer::
+
+    @property
+    def serialize(self):
+        return {
+            'node_properties': {
+                'sourceID': self.sourceID,
+                'name': self.name,
+                'country_codes': self.country_codes,
+                'valid_until': self.valid_until,
+                'countries': self.countries,
+                'node_id': self.node_id,
+            },
+        }
+
+Other::
+
+    @property
+    def serialize(self):
+        return{
+            'node_properties': {
+                'sourceID': self.sourceID,
+                'name': self.name,
+                'countries': self.countries,
+                'valid_until': self.valid_until,
+                'node_id': self.node_id,
+            },
+        }
+
+Call the serialize methods on the helpers
+=========================================
+
+Now instead of your ``fetch_nodes`` function on your ``fetch_api/utils.py`` looking like this::
+
+    def fetch_nodes(fetch_info):
+        node_type       = fetch_info['node_type']
+        search_word     = fetch_info['name']
+        country         = fetch_info['country']
+        limit           = fetch_info['limit']

```

Comment:
The limit and skip would be explained here or where? cause it would be good to explain why we are adding it since  , I believe, they do not know that we are fetching junks of 10

Reply:
Also, I refactored this helper to use python slice syntax which is more convenient.  Refer: https://github.com/neo4j-examples/paradise-papers-django/pull/50/files#diff-3faacf075dc77d7132127b7b416bcb89R66